admin/spam/views.py

```diff
@@ -126,23 +159,18 @@ def get_queryset(self):
         return Comment.find(query).sort(self.ordering)
 
     def get_context_data(self, **kwargs):
-        queryset = kwargs.pop('object_list', self.object_list)
```

Comment:
Django's ListView would handle all of this stuff for you if we didn't have to call serialize_comment... We could make a wrapper around the ODM queryset that calls serialize when something's actually accessed and then use that and your view code would end up with get_queryset and not much else.